src/main/java/com/inventor/email/service/ServicesE.java

```diff
@@ -0,0 +1,58 @@
+package com.inventor.email.service;
+
+import com.inventor.email.Email;
+import com.inventor.email.settings.EmailSettings;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ServicesE {
```

Comment:
Name of a component should indicate clear enough what this component is doing. Consider renaming this to `EmailService`